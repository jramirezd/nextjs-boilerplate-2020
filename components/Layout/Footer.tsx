import React from 'react';

import { footerText } from 'lib/constants';

import './Footer.scss';

const Footer = () => {
  return (
    <footer className="Footer">
      <div>{footerText}</div>
    </footer>
  );
};

export default Footer;
