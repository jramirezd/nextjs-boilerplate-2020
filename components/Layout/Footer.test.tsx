import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import expect from 'expect';
import enzymify from 'expect-enzyme';

import Footer from './Footer';

expect.extend(enzymify());

describe('Footer', () => {
  it('renders the Footer without errors', () => {
    const wrapper = shallow(<Footer />);
  });

  it('renders the footer as expected', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
