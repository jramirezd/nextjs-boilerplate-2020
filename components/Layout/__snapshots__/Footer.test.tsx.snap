// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders the footer as expected 1`] = `
<footer
  className="Footer"
>
  <div>
    Test footer
  </div>
</footer>
`;
